reanaconda/reanaconda.py

```diff
@@ -134,19 +134,30 @@ def __init__(self, qemu_args, append):
                                f'inst.updates=http://10.0.2.22 {append}']
         else:
             self.qemu_args += ['-append', 'inst.updates=http://10.0.2.22']
+        self.ssh = 'inst.sshd' in append
 
-    def run(self, http_port, loadvm=None):
+    def run(self, http_port, loadvm=None, second_time=False):
         self.monitor_port = find_free_port()
+        if second_time and self.ssh:
+            self.ssh_port = find_free_port()
+            ssh_hostfwd = f'hostfwd=tcp:127.0.0.1:{self.ssh_port}-:22,'
+        else:
+            ssh_hostfwd = ''
         cmd = ['qemu-system-x86_64'] + self.qemu_args + [
             '-monitor',
             f'tcp:127.0.0.1:{self.monitor_port},server,nowait,nodelay',
             '-device', 'virtio-net,netdev=net0', '-netdev',
-            'user,id=net0,'
+            f'user,id=net0,{ssh_hostfwd}'
             f'guestfwd=tcp:10.0.2.22:80-cmd:nc 127.0.0.1 {http_port}'
         ]
         if loadvm:
             cmd += ['-loadvm', loadvm]
         print(f'executing {cmd}')
+        if self.ssh and second_time:
+            print('to ssh into the VM, execute')
+            print('  ssh -o StrictHostKeyChecking=no '
+                  '-o UserKnownHostsFile=/dev/null '
+                  f'root@localhost -p {self.ssh_port}')
```

Comment:
This will connect you directly to tmux session.

```suggestion
                  f'anaconda@localhost -p {self.ssh_port}')
```